fuzzer/modules/oauth2_scopes/Dockerfile

```diff
@@ -1,9 +1,11 @@
-FROM python:3
+FROM fuzzer-base:latest
 
 WORKDIR /fuzzer
 
 COPY . /fuzzer/
 
+RUN cp -rf /fuzzer_base /fuzzer/api
+
 RUN python -m pip install -e /fuzzer/api
```

Comment:
I *think* we can actually just call

```
RUN python -m pip install -e /fuzzer_base
```

and not do the copying. Since we're installing the package it will be available globally as `import api`.

Reply:
I tried to do that....And it kept telling me "module not found". Not sure why, especially when running `pip list` shows all of the modules, the api one included.

Reply:
Hm, ok. I'm fine with this approach then if it works. Probably not worth the time to debug when we can be working on the presentation.